client/src/components/ResourcePreview.jsx

```diff
@@ -55,36 +57,43 @@ function ResourcePreview(props: Props) {
     setSrc(determineStockPhoto(category, subcategory));
   }, [category, setSrc, subcategory]);
 
-  const description = [];
+  let descriptionText = '';
+  if (description && description !== '') {
+    descriptionText += `${description} `;
+  }
+  const resourceInfo = [];
   let languages = '';
   availableLanguages.forEach((language) => {
     languages += `${language}, `;
   });
   if (languages !== '') {
     languages = languages.slice(0, languages.length - 2);
   }
-  let descriptionText = '';
+  let resourceInfoText = '';
   if (cost && cost !== '') {
-    descriptionText += `${cost} `;
+    resourceInfoText += `${cost} `;
   }
   if (city && city !== '') {
-    if (descriptionText !== '') {
-      descriptionText += '• ';
+    if (resourceInfoText !== '') {
+      resourceInfoText += '• ';
     }
-    descriptionText += `${city} `;
+    resourceInfoText += `${city} `;
   }
   if (languages && languages !== '') {
-    if (descriptionText !== '') {
-      descriptionText += '• ';
+    if (resourceInfoText !== '') {
+      resourceInfoText += '• ';
     }
-    descriptionText += `${languages} `;
+    resourceInfoText += `${languages} `;
   }
-  description.push(
-    <div key="description" style={{ color: 'black' }}>
-      {descriptionText}
-    </div>,
-  );
 
+  const descriptions = (
+    <>
+      <div className="description">{descriptionText}</div>,
```

Comment:
I think a comma was accidentally copied at the end of this line 😂 can it be removed?